src/userbase-server/app.js

```diff
@@ -526,6 +575,62 @@ exports.listUsersWithPagination = async function (req, res) {
   }
 }
 
+exports.listUsersForDatabaseWithPagination = async function (req, res) {
+  let logChildObject
+  try {
+    const databaseId = req.params.databaseId
+    const nextPageToken = req.query.nextPageToken
+
+    logChildObject = { ...res.locals.logChildObject, databaseId, nextPageToken }
+    logger.child(logChildObject).info('Listing users from Admin API')
+
+    _validateDatabaseId(databaseId)
+
+    // get database stuff
+    const db = await dbController.findDatabaseByDatabaseId(databaseId)
+    if (!db) throw {
+      status: statusCodes['Not Found'],
+      error: { message: 'Database not found.' }
+    }
+    const ownerId = db['owner-id']
+
+    // get user stuff from db owner
+    const owner = await userController.getUserByUserId(ownerId)
```

Comment:
Throw `Database not found.` if owner is deleted (either does not exist or has `deleted` set)

Reply:
Done